gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -93,12 +111,18 @@ def print_help(self):
     swaps         Recent swaps done on Uniswap
 [src][Defi Llama][/src]
     ldapps        Lists dApps
-    gdapps        Display top DeFi dApps grouped by chain
-    stvl          Display historical values of the total sum of TVLs from all dApps
-    dtvl          Display historical total value locked (TVL) by dApp
-[src][Terra Engineer][/src]
-    aterra        Displays 30-day history of specified asset in terra address
-    ayr           Displays 30-day history of anchor yield reserve[/cmds]
+    gdapps        Displays top DeFi dApps grouped by chain
+    stvl          Displays historical values of the total sum of TVLs from all dApps
+    dtvl          Displays historical total value locked (TVL) by dApp
+[src][Terra][/src]
```

Comment:
can you group commands by source and remove the source from the end of command as @DidierRLopes said